packages/admin-page/src/footer.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Automattic\Jetpack\AdminPage;
```

Comment:
Should our namespace be `Admin_Page` instead?

Reply:
I agree. Underscores are much more appropriate here.